### PR TITLE
fix: fix the error of pylint in demo directory

### LINKED ISSUE #1883: python: `pylint` check found many error
**Bug Description**
There are many error about `pylint`, and the pr of [feat: support pylint check in github action](https://github.com/4paradigm/OpenMLDB/pull/1873) has already integrate pylint to GitHub actions, So it is time to solve those error.

**Expected Behavior**
Execute `make lint` and got no errors.

**Steps to Reproduce**

1. execute command of `make lint`

And I'd like to solve this issue.

### PR DESCRIPTION
**Pull request by mangoGoForward**
Signed-off-by: mango <xu.weiKyrie@foxmail.com>

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix

* **What is the current behavior?** (You can also link to an open issue here)
Part resolved #1883

* **What is the new behavior (if this is a feature change)?**
None

### CODE REVIEW

File: pylintrc [hunk 1]
```diff
@@ -89,7 +89,9 @@ disable=abstract-method,
         long-suffix,
         map-builtin-not-iterating,
         misplaced-comparison-constant,
+        missing-class-docstring,
```
Comment (aceforeverd) [inline pylintrc]: class and module doc string should be available

**Comment by aceforeverd** [comment]
@vagetablechicken have a refactor in #1902, would u pls rework after that ? Currently you can just ignore `predict_server.py` and `train_and_serve.py`

**Comment by mangoGoForward** [comment]
> 

OK.

**Comment by aceforeverd** [comment]
> > 
> 
> OK.

I think the two file should be fine after merge. If you are interested in styling from pylint, feel free to comment in #1902

File: pylintrc (hunk 1 above)
Comment (mangoGoForward) [reply pylintrc]: Done.

**Comment by mangoGoForward** [comment]
> @vagetablechicken have a refactor in #1902, would u pls rework after that ? Currently you can just ignore `predict_server.py` and `train_and_serve.py`

The refactor in #1902 has been merged, so the modification of the two files also committed in this pr.

File: demo/talkingdata-adtracking-fraud-detection/train_and_serve.py [hunk 2]
```diff
@@ -89,86 +90,84 @@ def xgb_modelfit_nocv(params, dtrain, dvalid, predictors, target='target', objec
...
-connection.execute("SET @@job_timeout=1200000;")
-connection.execute("{} INTO OUTFILE '{}' OPTIONS(mode='overwrite');".format(
-    sql_part, train_feature_dir))
+connection.execute('SET @@job_timeout=1200000;')
+connection.execute(f"{sql_part} INTO OUTFILE '{train_feature_dir}' OPTIONS(mode='overwrite');")
 
-print("Load features from feature dir {}".format(train_feature_dir))
+print(f'Load features from feature dir {train_feature_dir}')
 # train_feature_dir has multi csv files
 train_df = pd.concat(map(lambda file: pd.read_csv(file), glob.glob(
```
Comment (aceforeverd) [inline demo/talkingdata-adtracking-fraud-detection/train_and_serve.py]: rm the lambda. 
```python
map(pd.read_csv, glob.glob(.....
```

File: demo/talkingdata-adtracking-fraud-detection/train_and_serve.py [hunk 3]
```diff
@@ -89,86 +90,84 @@ def xgb_modelfit_nocv(params, dtrain, dvalid, predictors, target='target', objec
...
 
 def nothrow_execute(sql):
     # only used for drop deployment, cuz 'if not exist' is not supported now
     try:
-        print("execute " + sql)
-        ok, rs = connection.execute(sql)
+        print('execute ' + sql)
+        _, rs = connection.execute(sql)
         print(rs)
     except Exception as e:
```
Comment (aceforeverd) [inline demo/talkingdata-adtracking-fraud-detection/train_and_serve.py:112]: too general exception.
add a magic comment to ignore this temporally. 

same in `demo/predict-taxi-trip-duration/test/import.py`

File: demo/talkingdata-adtracking-fraud-detection/predict_server.py [hunk 4]
```diff
@@ -165,7 +167,7 @@ def make_app():
     global global_args
```
Comment (aceforeverd) [inline demo/talkingdata-adtracking-fraud-detection/predict_server.py]: global at module level.
this global stmt can be just removed

**Review by aceforeverd — CHANGES_REQUESTED** [review]

File: demo/predict-taxi-trip-duration/test/import.py [hunk 5]
```diff
@@ -42,26 +39,29 @@
 engine = db.create_engine('openmldb:///db_test?zk=127.0.0.1:2181&zkPath=/openmldb')
 connection = engine.connect()
 try:
-    connection.execute("create database db_test;");
+    connection.execute('create database db_test;')
 except Exception as e:
```
Comment (aceforeverd) [inline demo/predict-taxi-trip-duration/test/import.py:44]: add magic comment to ignore this

File: demo/talkingdata-adtracking-fraud-detection/train_and_serve.py (hunk 2 above)
Comment (mangoGoForward) [reply demo/talkingdata-adtracking-fraud-detection/train_and_serve.py]: done.

File: demo/talkingdata-adtracking-fraud-detection/train_and_serve.py (hunk 3 above)
Comment (mangoGoForward) [reply demo/talkingdata-adtracking-fraud-detection/train_and_serve.py:112]: done.

File: demo/talkingdata-adtracking-fraud-detection/predict_server.py (hunk 4 above)
Comment (mangoGoForward) [reply demo/talkingdata-adtracking-fraud-detection/predict_server.py]: done.

**Review by aceforeverd — APPROVED** [review]

File: demo/talkingdata-adtracking-fraud-detection/predict_server.py [hunk 6]
```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Module of docstring"""
```
Comment (vagetablechicken) [inline demo/talkingdata-adtracking-fraud-detection/predict_server.py]: doc string?

File: demo/talkingdata-adtracking-fraud-detection/predict_server.py [hunk 7]
```diff
@@ -60,18 +61,19 @@ def request_row_cvt(json_row):
...
 def get_result(response):
     result = json.loads(response.text)
-    logging.info('request result: %s', result)
-    return result['data']['data']
+    logging.info("request result: %s", result)
+    return result["data"]["data"]
 
 
 class PredictHandler(tornado.web.RequestHandler):
+    """Class PredictHandler docstring."""
```
Comment (vagetablechicken) [inline demo/talkingdata-adtracking-fraud-detection/predict_server.py]: docstring?

File: demo/talkingdata-adtracking-fraud-detection/predict_server.py [hunk 8]
```diff
@@ -162,10 +164,9 @@ def make_app():
                         default=True)
     args = parser.parse_args()
 
-    global global_args
     global_args = vars(args)
     print(global_args)
-    logging.info('init args: %s', global_args)
+    logging.info("init args: %s", global_args)
```
Comment (vagetablechicken) [inline demo/talkingdata-adtracking-fraud-detection/predict_server.py:169]: Why use '' in train_and_serve.py, and use "" here? This file can use '' too.

File: demo/talkingdata-adtracking-fraud-detection/train_and_serve.py [hunk 9]
```diff
@@ -1,37 +1,38 @@
+"""Module of docstring"""
```
Comment (vagetablechicken) [inline demo/talkingdata-adtracking-fraud-detection/train_and_serve.py]: comment

File: demo/talkingdata-adtracking-fraud-detection/predict_server.py (hunk 8 above)
Comment (mangoGoForward) [reply demo/talkingdata-adtracking-fraud-detection/predict_server.py:169]: As the comment of `pylint`(https://github.com/PyCQA/pylint/blob/34c6aede493cfc72dcc94ae9085ab68b49e2960c/pylint/checkers/strings.py#L647-L651), the quote delimiters should used consistently throughout a module, may the rest of  `train_and_serve ` has more `''`, but the file of `predict_server.py` has more `'`, I think just keep consistent in whole file is ok.

File: demo/talkingdata-adtracking-fraud-detection/train_and_serve.py (hunk 9 above)
Comment (mangoGoForward) [reply demo/talkingdata-adtracking-fraud-detection/train_and_serve.py]: Done.

File: demo/talkingdata-adtracking-fraud-detection/predict_server.py (hunk 8 above)
Comment (vagetablechicken) [reply demo/talkingdata-adtracking-fraud-detection/predict_server.py:169]: Only ~200 lines, small file, use `'` is ok. That's up to you

File: demo/talkingdata-adtracking-fraud-detection/train_and_serve.py [hunk 10]
```diff
@@ -1,13 +1,14 @@
+"""Module of training and request to predict server"""
 import gc
 import os
 import time
 import glob
 import requests
 # fmt:off
-import openmldb
```
Comment (vagetablechicken) [inline demo/talkingdata-adtracking-fraud-detection/train_and_serve.py:7]: don't delete `import openmldb`. It may get seg fault in some machine

Comment (mangoGoForward) [reply demo/talkingdata-adtracking-fraud-detection/train_and_serve.py:7]: Done.

File: demo/talkingdata-adtracking-fraud-detection/predict_server.py (hunk 8 above)
Comment (mangoGoForward) [reply demo/talkingdata-adtracking-fraud-detection/predict_server.py:169]: done.

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**